js/rxjs.md

```diff
@@ -0,0 +1,24 @@
+# Bloomfire RxJS Style Guide
+
+*Very much a work in progress*
+
+## Table of Contents
+
+  1. [Observable Naming](#observables)
+
+## observables 
+
+  - Files containing observables should be named in camelCase.
+  - Observable classes should be capitalized and MixedCase.
+  - Observable classes should substitute a trailing $ for the word "observable."
+  - Observable instances should be camelCase.
+
+  ```jsx
+  import SomeAPICall$ from 'app/observables/someAPICallObservable';
+  ````
+
+  ```jsx
+  const someAPICall = app.getInstance(SomeAPICall$);
```

Comment:
So basically this suggestion is:
```
file: someObservable
class: some$
instance: some
```
but from what I've seen, the $ is for the *instance*, since that tells the developer that the object in question has the requisite Rx operators. So perhaps:
```
file: someObservable
class: some
instance: some$
```
This is up for discussion of course. I'm not sure of the best way, since the whole app.getInstance thing we do adds that  extra distinction between class and instance.